radis/test/lbl/test_loader.py

```diff
@@ -222,13 +222,175 @@ def test_custom_abundance(verbose=True, plot=False, *args, **kwargs):
+        "hitemp", memory_mapping_engine="vaex", output="vaex", load_columns="all"
+    )
+    df1 = sf.df0
+
+    # Fetching in Pandas dataframe format
+    sf.dataframe_type = "pandas"
+    sf.fetch_databank(
+        "hitemp", memory_mapping_engine="vaex", output="pandas", load_columns="all"
+    )
+    df2 = sf.df0
+
+    # Comparing the dataframe fetched from HITEMP
+    assert compare_dataframe(df1, df2, df2.columns)
+
+    # EXOMOL
+    # Comparing Vaex dataframe and Pandas dataframe fetched from EXOMOL
+
+    # Fetching in vaex dataframe format
+    sf.dataframe_type = "vaex"
+    sf.fetch_databank(
+        "exomol", memory_mapping_engine="vaex", output="vaex", load_columns="all"
+    )
+    df1 = sf.df0
+
+    # Fetching in Pandas dataframe format
+    sf.dataframe_type = "pandas"
+    sf.fetch_databank(
+        "exomol", memory_mapping_engine="vaex", output="pandas", load_columns="all"
+    )
+    df2 = sf.df0
+
+    # Comparing the dataframe fetched from EXOMOL
+    assert compare_dataframe(df1, df2, df2.columns)
+
+    # GEISA
+    # Comparing Vaex dataframe and Pandas dataframe fetched from GEISA
+
+    # Fetching in vaex dataframe format
+    sf.dataframe_type = "vaex"
+    sf.fetch_databank(
+        "geisa", memory_mapping_engine="vaex", output="vaex", load_columns="all"
+    )
+    df1 = sf.df0
+
+    # Fetching in Pandas dataframe format
+    sf.dataframe_type = "pandas"
+    sf.fetch_databank(
+        "geisa", memory_mapping_engine="vaex", output="pandas", load_columns="all"
+    )
+    df2 = sf.df0
+
+    # Comparing the dataframe fetched from GEISA
+    assert compare_dataframe(df1, df2, df2.columns)
+
+
+def test_vaex_and_pandas_dataframe_load_databank():
+    import numpy as np
+
+    from radis import SpectrumFactory
+
+    # Comparing HITRAN
+    sf = SpectrumFactory(
+        wavenum_min=2380,
+        wavenum_max=2400,
+        mole_fraction=400e-6,
+        path_length=100,  # cm
+        isotope=[1],
+    )
+
+    # Loading in vaex dataframe format
+    sf.load_databank("HITRAN-CO2-TEST", output="vaex")
+    df1 = sf.df0
+
+    # Loading in  pandas dataframe format
+    sf.load_databank("HITRAN-CO2-TEST")
+    df2 = sf.df0
+
+    # Comparing both the dataframes
+    for column in df2.columns:
+        assert np.all(df1[column].to_numpy() == df2[column])
+
+    # Comparing HITEMP
+    sf = SpectrumFactory(
+        2284.2,
+        2284.6,
+        wstep=0.001,  # cm-1
+        pressure=20 * 1e-3,  # bar
+        cutoff=0,
+        path_length=0.1,
+        mole_fraction=400e-6,
+        molecule="CO2",
+        isotope="1,2",
+        medium="vacuum",
+        truncation=5,
+        verbose=0,
+    )
+    sf.warnings["MissingSelfBroadeningWarning"] = "ignore"
+
+    # Loading in vaex dataframe format
+    sf.load_databank("HITEMP-CO2-TEST", output="vaex", load_columns="all")
+    df1 = sf.df0
+
+    # Loadint in pandas dataframe format
+    sf.load_databank("HITEMP-CO2-TEST", load_columns="all")
```

Comment:
```suggestion
    sf.load_databank("HITEMP-CO2-TEST", output="pandas", load_columns="all")
```
Learning : in tests never assume the default value of a parameter. Be explicit. Default might change in the future